ansible/roles/dev_machine/files/vs_code_setup.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Copyright 2023 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+sr_build_tools_branch="F_%23SP-476_vs_code_setup_lint"
```

Comment:
I guess we would need to change this before we can merge.

Reply:
Addressed in https://github.com/shadow-robot/sr-build-tools/pull/697/commits/89c8670d4f1388b9f4219295f7f553aa6b8b76ed